server.ts

```diff
@@ -15,12 +17,31 @@ app
   .use(helmet())
   .use(morgan("dev"))
   .use(express.json())
-  .use(express.urlencoded({ extended: true }));
+  .use(express.urlencoded({ extended: true }))
+  .use(
+    cookieSession({
+      name: "session",
+      keys: [
+        process.env.COOKIE_SECRET
+          ? process.env.COOKIE_SECRET
+          : "supersecretkey",
+      ],
+      maxAge: 2 * 24 * 60 * 60 * 1000,
+      secure: process.env.NODE_ENV === "dev" ? false : true,
+      httpOnly: process.env.NODE_ENV === "dev" ? false : true,
```

Comment:
`secure: process.env.NODE_ENV !== "dev"` instead of the whole thing :)

and same for `httpOnly`